src/server/rdb_load.cc

```diff
@@ -211,6 +213,22 @@ bool resizeStringSet(robj* set, size_t size, bool use_set2) {
   }
 }
 
+void RunOnceAsCommand(Service* service, const CommandId* cid,
```

Comment:
This is a useful utility, consider adding it as a static function of `Transaction`